test_zscroll.py

```diff
@@ -37,7 +37,7 @@ def test_make_visual_len(string, new_length, new_string):
     ('false', ''),
 ])
 def test_shell_output(command, result):
-    assert z.shell_output(command) == result
+    assert z.shell_output(command, False) == result
```

Comment:
If possible, could you add tests with something simple like "$(echo hello)"? It looks like I will have to re-setup CI for testing. I think this should work on CI but haven't tried yet.